### DISCUSSION TITLE
[PATTERN] Add a star pattern in Python, Pattern1

### DISCUSSION THREAD

**Discussion by vishaldev21** [Vishal21121/Patterns-different-language#147] [General]
### Description

Task:
for the given input n the following pattern should print (n=5 for the below pattern):

### Pattern Image

![1](https://github.com/Vishal21121/Patterns-different-language/assets/89929777/34159959-e94e-4d74-8c21-c9dfc564822e)

### Filename

Pattern1.py

**Comment by RaghavLimbani** [comment]
can you assign me this pattern issue I think I can solve it

**Comment by vishaldev21** [comment]
ok, @RaghavLimbani you can work on this issue.

**Comment by RaghavLimbani** [comment]
sure

**Comment by RaghavLimbani** [comment]
@Vishal21121  I have forked the repo and pushed the changes but its not showing in your PRs, let me correct myself

**Comment by vishaldev21** [comment]
@RaghavLimbani the pr was done in your own forked repo merge that pr and then make a pr in this repo.

**Comment by RaghavLimbani** [comment]
okay

**Comment by RaghavLimbani** [comment]
@Vishal21121 can you help me with it am kind of not quite not getting the concept

**Reply by vishaldev21** [reply]
ok @RaghavLimbani can you explain what issue you are facing?

**Comment by RaghavLimbani** [comment]
I have commited and merged changes to the forked repo now how to push the changes to your repo?

**Reply by vishaldev21** [reply]
There will be an option that your repo is 1 commit ahead of this repo click on that and then you will get the option to make pr.

**Comment by RaghavLimbani** [comment]
I was learning about it, I think I will be making pull request maybe today or tommorow

**Reply by vishaldev21** [reply]
ok, @RaghavLimbani take your time and If you need any help you can ask me.

**Comment by RaghavLimbani** [comment]
@Vishal21121 I have made a pull request with Branch name Raghav, sorry for that mistake, and I have made mistakes in some previous commits so some commits might contain the wrond data so please review my code carefully, thanks for the help and suggestions. Have a good day!
